core/src/main/java/org/mule/runtime/core/internal/processor/AsyncDelegateMessageProcessor.java

```diff
@@ -133,6 +143,27 @@ public void initialise() throws InitialisationException {
     super.initialise();
   }
 
+  /**
+   * A fallback method for creating a {@link ProcessingStrategyFactory} to be used in case the user hasn't specified one through
+   * either , through {@link MuleConfiguration#getDefaultProcessingStrategyFactory()} or the {@link ProcessingStrategyFactory}
+   * class name system property
+   *
+   * @return a {@link DirectProcessingStrategyFactory}
+   */
+  protected ProcessingStrategyFactory createDefaultProcessingStrategyFactory() {
+    return new DirectProcessingStrategyFactory();
+  }
+
+  private ProcessingStrategyFactory defaultProcessingStrategy() {
+    final ProcessingStrategyFactory defaultProcessingStrategyFactory =
```

Comment:
Will this change? Might be better to store it.

Reply:
shouldn't change, that default can only be set via system property